src/main/java/uk/gov/hmcts/reform/hmc/service/OperationalReportsService.java

```diff
@@ -13,11 +13,11 @@ public interface OperationalReportsService {
 
     File createCsvDataForAwaitingActuals() throws IOException;
 
-    List<HearingRequestForCsv> createCsvObjectsForGivenStatuses(List<String> statuses);
+    List<HearingRequestForCsv> createCsvObjectsForGivenStatuses(List<String> statuses, boolean isError);
```

Comment:
isError flag not needed and can be removed in all the files

Reply:
isError flag has been removed in latest commit